crates/ruff_python_parser/tests/snapshots/invalid_syntax@statements__match__as_pattern_3.py.snap

```diff
@@ -118,7 +118,7 @@ Module(
 2 |     #     Not in the mapping start token set, so the list parsing bails
 3 |     #     v
 4 |     case {(x as y): 1}:
-  |                      ^ Syntax Error: Expected a statement
+  |                      ^ Syntax Error: expected Dedent, found Rbrace
 5 |         pass
```

Comment:
Not a good recovery. I think this is because of an invalid use of `as` pattern from which it's difficult to recover.